### PR TITLE
Feature/search restyle

### PR DESCRIPTION
**Pull request by silksil**
- include search component on search and home page
- include functionality to  set-up forms  using react-hook-form and yup (see MUI template for examples)
- pass query params when going from home page to search page

### CODE REVIEW

File: src/containers/FaqAccordion.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+const faq = [
+  {
+    question: 'What is WIN.so?',
+    answer:
+      'Win.so is a web 2.5 accommodation booking platform created by Winding Tree and Simard teams. It operates on smart contract technology and uses crypto payments.'
+  },
+  {
+    question: 'What is Winding Tree?',
+    answers:
```
Comment (bobetbat) [inline src/containers/FaqAccordion.tsx:22]: This is the point I mentioned on call regarding loosing commits. This file was fixed in previous commit, should be `answer`

File: src/containers/FaqAccordion.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  },
+  {
+    question: 'How to report issues?',
+    answer:
+      'If you have found any issues while booking or using our platform, please provide us with the details and print screens to improve@win.so'
+  }
+];
+
+export const FaqAccordion = () => {
+  return (
```
Comment (bobetbat) [inline src/containers/FaqAccordion.tsx:101]: some closing logic used to be here, check out the last main please

**Comment by bobetbat** [comment]
would be good to refetch only once after bad response (3 is to much)
Screenshot 2022-08-24 at 12 15 28

**Review by bobetbat — DISMISSED** [review]
LGTM

**Review by valera-rozuvan — APPROVED** [review]

**Comment by valera-rozuvan** [comment]
LGTM

**Pull request merged**